source/includes/fundamentals/code-snippets/Geo.java

```diff
@@ -0,0 +1,78 @@
+package com.mycompany.app;
+
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import org.bson.Document;
+
+// begin exampleImports
+import java.util.Arrays;
+import org.bson.conversions.Bson;
+import com.mongodb.client.model.geojson.Point;
+import com.mongodb.client.model.geojson.Polygon;
+import com.mongodb.client.model.geojson.Position;
+import static com.mongodb.client.model.Filters.near;
+import static com.mongodb.client.model.Filters.geoWithin;
+import static com.mongodb.client.model.Projections.fields;
+import static com.mongodb.client.model.Projections.include;
+import static com.mongodb.client.model.Projections.excludeId;
+// end exampleImports
+
+public class Geo {
+
+    private final MongoClient mongoClient;
+
+    public Geo() {
+        final String uri = System.getenv("DRIVER_REF_URI");
+        mongoClient = MongoClients.create(uri);
+
+    }
+
+    public static void main(String[] args) {
+        Geo geo = new Geo();
+        geo.go();
+    }
+
+    public void go() {
+        Geo geo = new Geo();
+        System.out.println("Near Example: ");
+        geo.nearExample();
+        System.out.println("Range Example: ");
+        geo.rangeExample();
+
+    }
+
+    private void nearExample() {
+        // begin findExample
+
+        // code to set up your mongo client ...
+        MongoDatabase database = mongoClient.getDatabase("sample_mflix");
+        MongoCollection<Document> collection = database.getCollection("theaters");
+        final Point centralPark = new Point(new Position(-73.9667, 40.78));
```

Comment:
Nit: let's be consistent about our use of `final` for local variables.  IIRC we have not been using them, and I think for examples like this they are a distraction.  So recommend to remove `final` from local variables in all examples.

Reply:
Thanks for the note. Good catch, I agree that `final` does not make this example more clear. Going forward, I will not use `final` for local variables in examples for the sake of consistency and clarity. 

Fix: https://github.com/mongodb/docs-java/pull/78/commits/b70de87befe875157de0372803977a1d8d8efe09